python/ray/data/dataset.py

```diff
@@ -2462,6 +2462,7 @@ def to_tf(
         prefetch_blocks: int = 0,
         batch_size: int = 1,
         drop_last: bool = False,
+        unsqueeze_label_tensor: bool = False,
```

Comment:
`Dataset.to_torch` has `unsqueeze_label_tensor: bool = True`, but I'm defaulting to `False` here for backwards compatibility.